litmus-portal/cluster-k8s-manifest.yml

```diff
@@ -299,9 +299,9 @@ rules:
     verbs: [create, get, watch, patch, delete, list]
 
   # for tracking & getting logs of the pods created by workflow controller to implement individual steps in the workflow
-  - apiGroups: [""]
-    resources: [pods/log]
-    verbs: [get, watch]
+  - apiGroups: ['']
+    resources: ['pods/log', 'secrets', 'configmaps']
```

Comment:
Can you please remove inverted commas from verbs and resources. Refer to the above Rule.
( Cluster-k8s-manifest.yml,litmus-portal-dev-manifest-template.yml and namespaced-k8s-manifest.yml)